### ISSUE TITLE
Organization level webhook

### ISSUE THREAD

**Issue by ColinKotar** [deep-compute/gitkanban#177]

**Comment by ColinKotar** [comment]
Right now there seems to be no way of keeping track of new repositories being created, since, from my understanding, it requires an Organization level webhook. To avoid being pinged too often by an organization level webhook, we can easily configure the webhook to only POST the most important changes. Also, for the time being we can manually add a webhook that only keeps track of new repositories being made.

**Comment by prashanthellina** [comment]
@ColinKotar Makes sense. We could make an org level webhook that posts purely only repo creation, deletion events and nothing else.
